mindsdb/integrations/handlers/gcs_handler/README.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Google Cloud Storage
+sidebarTitle: Google Cloud Storage
+---
+
+This documentation describes the integration of MindsDB with [Google Cloud Storage](https://cloud.google.com/storage), an object storage service that offers industry-leading scalability, data availability, security, and performance.
+
+## Prerequisites
+
+1. Before proceeding, ensure that MindsDB is installed locally via [Docker](/setup/self-hosted/docker) or [Docker Desktop](/setup/self-hosted/docker-desktop).
+2. Install the dependencies from the `requirements.txt` file. Run the following command
```

Comment:
Can you please replace this second step with what is used throughout our documentation? You can refer [this](https://docs.mindsdb.com/integrations/data-integrations/google-bigquery).

Reply:
Done